realsense2_camera/launch/rs_launch.py

```diff
@@ -46,23 +47,32 @@
                            {'name': 'enable_fisheye1',              'default': 'true', 'description': 'enable fisheye1 stream'},
                            {'name': 'enable_fisheye2',              'default': 'true', 'description': 'enable fisheye2 stream'},
                            {'name': 'enable_confidence',            'default': 'true', 'description': 'enable confidence'},
-                           {'name': 'gyro_fps',                     'default': '0', 'description': "''"},                           
-                           {'name': 'accel_fps',                    'default': '0', 'description': "''"},                           
-                           {'name': 'enable_gyro',                  'default': 'false', 'description': "'enable gyro stream'"},                           
-                           {'name': 'enable_accel',                 'default': 'false', 'description': "'enable accel stream'"},                           
-                           {'name': 'enable_pose',                  'default': 'true', 'description': "'enable pose stream'"},                           
-                           {'name': 'pose_fps',                     'default': '200', 'description': "''"},                           
+                           {'name': 'gyro_fps',                     'default': '0', 'description': "''"},
+                           {'name': 'accel_fps',                    'default': '0', 'description': "''"},
+                           {'name': 'enable_gyro',                  'default': 'false', 'description': "''"},
+                           {'name': 'enable_accel',                 'default': 'false', 'description': "''"},
+                           {'name': 'enable_pose',                  'default': 'true', 'description': "''"},
+                           {'name': 'pose_fps',                     'default': '200', 'description': "''"},
+                           {'name': 'pointcloud.enable',            'default': 'false', 'description': ''},
+                           {'name': 'gyro_fps',                     'default': '0', 'description': "''"},
+                           {'name': 'accel_fps',                    'default': '0', 'description': "''"},
+                           {'name': 'enable_gyro',                  'default': 'false', 'description': "'enable gyro stream'"},
```

Comment:
I think you had some conflicts, that were not merged perfectly.
Multiple parameters are duplicated (some without description, and some with)
E.g., enable_gyro is in two places in rs_launch.py

Reply:
opps there was duplications :)
sovled now